src/test/java/codeu/model/data/UserTest.java

```diff
@@ -27,12 +28,14 @@ public void testCreate() {
     String name = "test_username";
     String passwordHash = "$2a$10$bBiLUAVmUFK6Iwg5rmpBUOIBW6rIMhU1eKfi3KR60V9UXaYTwPfHy";
     Instant creation = Instant.now();
+    Instant lastLogin = Instant.now();
 
     User user = new User(id, name, passwordHash, creation);
 
     Assert.assertEquals(id, user.getId());
     Assert.assertEquals(name, user.getName());
     Assert.assertEquals(passwordHash, user.getPasswordHash());
     Assert.assertEquals(creation, user.getCreationTime());
+  //  Assert.assertEquals(lastLogin, user.getLastLogin());
```

Comment:
The test here mostly test the creation of the user, which is probably not so related to the last active time. 

You can have a test that specifically test the lastLogin after the user logs in. The test may be in the controller instead.